src/utils.ts

```diff
@@ -152,6 +165,27 @@ export function createSegments(bucket: any, mainColor: string, isHighlight: bool
     }));
 }
 
+
+
+/**
+ * Creates a timeseries data for facet sparkline.
```

Comment:
comment nitpick: since "data" is plural, consider "Creates timeseries data for a facet sparkline"